### PR TITLE
fix: scripts/get-task-fuzzy-duration.py: Account for long running tasks

### PR DESCRIPTION
**Pull request by pablomh**
Instead of waiting an arbitrary amount of time, look for tasks that doesn't seem to progress in a certain amount of time (in this case: 10 minutes).

### CODE REVIEW

File: scripts/get-task-fuzzy-duration.py [hunk 1]
```diff
@@ -97,14 +97,27 @@ def print_result(output_format, data):
 
 def investigate_task(args):
     time_before = time.time()
+    parent_task = get_json(
+        args.hostname, "/foreman_tasks/api/tasks/%s" % args.task_id,
+        args.username, args.password)
+    finished_count_before = parent_task['output']['success_count'] + parent_task['output']['failed_count']
+    timeout_counter = 0
+    timeout_minutes = 10
```
Comment (jhutar) [inline scripts/get-task-fuzzy-duration.py:105]: Could you repurpose `args.timeout` for `timeout_minutes` (and change default)?

File: scripts/get-task-fuzzy-duration.py [hunk 2]
```diff
@@ -97,14 +97,27 @@ def print_result(output_format, data):
 
 def investigate_task(args):
     time_before = time.time()
+    parent_task = get_json(
```
Comment (jhutar) [inline scripts/get-task-fuzzy-duration.py:100]: What about changing a logic of that `while True` internals so we do not need two of this `get_json(...)` calls here?

**Review by jhutar — CHANGES_REQUESTED** [review]
Feel free to merge even in this state. Comments I made are not important.

**Comment by pablomh** [comment]
I included by mistake this commit in a bunch with other. I'll integrate your comments in a subsequent revision :/

**Pull request merged**